PracticeGit/Program.c

```diff
@@ -1,13 +1,50 @@
-// ------------------------------------------------------------------------------------------------
-// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
-// Copyright (c) Metamation India.
-// ------------------------------------------------------------------
-// Program.c
-// Program on main branch.
-// ------------------------------------------------------------------------------------------------
+#include <string.h>
+#include <ctype.h>
+#include <limits.h>
+#include "Program.h"
 
-#include <stdio.h>
+int PalindromeChecker (const char* str) {
+   size_t left = 0;
```

Comment:
You can just have a for loop and check till str.length/2. Also you can just use indexing in this for loop to check.